04/app.js

```diff
@@ -1 +1,4 @@
-const years = [1980, 1934, 2002, 2019];
\ No newline at end of file
+const years = [1980, 1934, 2002, 2019];
+
+const passYears = years.map(function(element){ return 2022 - element});
```

Comment:
Pisanie tego w jednej linii w mojej ocenie jest nieczytelne - lepiej to zrobić w kilku liniach :)